src/pages/NewsItem/NewsItem.tsx

```diff
@@ -38,16 +38,31 @@ export default function NewsItem() {
 						<a
 							href={newsItem.url}
 							target="_blank"
-							className="mb-6"
+							className="mb-2"
 							rel="noreferrer"
 						>
 							Read everything
 						</a>
-						<div className="mb-6 mt-auto flex justify-between gap-4 text-sm text-gray-500">
+						<div className="mb-4 mt-auto flex justify-between gap-4 text-sm text-gray-500">
 							<p className="self-end whitespace-nowrap">
```

Comment:
maybe this can be https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time